### ISSUE TITLE
[Request] Support for the opus audio codec.

### ISSUE THREAD

**Issue by mkditto** [karimknaebel/phonograph-issue-tracker#85]
Could you add support for the opus audio codec please? It's main benefit is that at low bitrates it has audio quality that rivals much higher bitrate mp3 files. It would be super awesome especially for saving space. Thanks!

**Comment by karimknaebel** [comment]
No, Phonograph uses Androids inbuilt media player api

**Comment by kiuuik** [comment]
Another android foss media player named "Vanilla Music" can play my test opus file on my Nexus with Android 7. 
Afaik Vanilla also relies solely on built-in android media functions.
A developer there mentions that it [works on his Android6 device quite well whereas it's not working on Android5](https://github.com/vanilla-music/vanilla/issues/384)
I created my test file using "ffmpeg -i input.mp3 output.opus". Then I renamed "output.opus" to "output.ogg" and put it in my music folder. Vanilla can play it, add it to a playlist and seems to read my tags (though I only use artist and track).
When I browse to the folder with output.ogg in Phonograph it's not shown.

**Comment by karimknaebel** [comment]
@kiuuik can you find it with Phonographs folder browser?

**Comment by kiuuik** [comment]
I don't know if this answers your question: 

**In Phonograph Music Player** (from Google play, version 0.13.3) after clicking on the Hamburger menu in the topleft corner I chose "Folders" from sidbar. Then I navigated to the folder containing my opus-ogg files. Then I don't see any files. I only see "empty". 

I tested again **in Vanilla Music** (from Google Play, version 1.0.43) with 10 files. I converted with "ffmpeg -i input.mp3 -vn -acodec libopus output.ogg". In Vanilla all opus-ogg files seem to be recognised and even the tags I used (this time I used artist, album, and track) are read and appear in the Artists, Albums, and Songs view. I also put all of these opus-ogg files in a playlist which I can also play from the Playlist view. All in all these files seem to suffice for basic functionality.

I tested on android 7.0 on my nexus.
